console/src/main/webapp/manager/app/app.es6

```diff
@@ -73,18 +73,43 @@ class AppController {
 }
 
 class StandaloneController {
-  static $inject = ['$scope', 'Orgs', 'User']
+  static $inject = ['$injector', '$window', '$scope', 'Orgs', 'User']
 
-  constructor ($scope, Org) {
-    if (!window.org) {
+  constructor ($injector, $window, $scope, Org) {
+    if (!$window.org) {
       $scope.org = new Org()
       return
     }
 
-    $scope.org = window.org
-    $scope.users = window.org.members
-    $scope.isReferentOrSuperUser = window.isReferentOrSuperUser
-    $scope.gdprAllowAccountDeletion = window.gdprAllowAccountDeletion
+    this.$injector = $injector
+
+    $scope.org = $window.org
+    $scope.users = $window.org.members
+    $scope.isReferentOrSuperUser = $window.isReferentOrSuperUser
+    $scope.gdprAllowAccountDeletion = $window.gdprAllowAccountDeletion
+    const CONSOLE_BASE_PATH = this.$injector.get('CONSOLE_BASE_PATH')
+    $scope.deleteURI = CONSOLE_BASE_PATH + 'account/gdpr/delete'
+
+    const i18n = {}
+    this.$injector.get('translate')('editUserDetailsForm.deleteConfirm', i18n)
+    this.$injector.get('translate')('editUserDetailsForm.deleteFail', i18n)
+
+    $scope.deleteUser = function () {
+      var isConfirmed = $window.confirm(i18n.deleteConfirm)
+
+      if (!isConfirmed) return false
+      $window.fetch($scope.deleteURI, { method: 'POST' })
+        .then(function (response) {
+          if (response.ok) {
+            $window.location.href = '/logout'
+          } else {
+            $window.alert(i18n.deleteFail)
+          }
+        })
+        .catch(function () {
+          $window.alert(i18n.deleteFail)
+        })
+    }
```

Comment:
Now that you move this code into a place where it's transpiled, you could save some few chars using arrow functions.

Reply:
Sure. I considered it was time to stop fussing around, but there are still lots of improvements to do: probably use $http.post instead of fetch for instance. And indeed, arrow functions

Reply:
Anyway, it looks that there would be some optimization to do. The editUserDetailsForm.jsp is some strange mix between jsp form, jquery and angular... I'd rather use one system for all of the form...

Reply:
@tonio  Following this discussion, I've optimized a bit the code. OK for you ?